src/main/java/mate/jdbc/dao/CarDao.java

```diff
@@ -0,0 +1,19 @@
+package mate.jdbc.dao;
+
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.model.Car;
+
+public interface CarDao {
+    Car create(Car car);
+
+    Optional<Car> get(Long id);
+
+    List<Car> getAll();
+
+    Car update(Car car);
+
+    List<Car> getAllByDriver(Long driverId);
```

Comment:
Consider renaming the method 'getAllByDriver' to 'getCarsByDriverId' for better clarity and understanding of the method's purpose 😊